.github/.gitignore

```diff
@@ -0,0 +1 @@
+*.html
```

Comment:
In what context is this file used?

Reply:
I've read through the documentation for [`pkgdown::build_home()`](https://pkgdown.r-lib.org/reference/build_home.html), one of the functions called by [`pkgdown::build_site()`](https://github.com/r-lib/pkgdown/blob/master/R/deploy-site.R#L172-L175) in GitHub Actions.

I believe that according to pkgdown conventions, `.github/` is used to hold several `.md` files that are subsequently rendered to `.html` and linked to. Some examples are `CONTRIBUTING.md` (currently in the root folder, which pkgdown also detects), `CODE_OF_CONDUCT.md`, etc. The `.github/.gitignore` file is so that the `.html` files are not added to the commit tree. We could move your `CONTRIBUTING.md` into `.github/` if you're interested, before merging the PR.

Reply:
Thanks for the explanation! `CONTRIBUTING.md` already shows up in the sidebar, so I wouldn't move it around for now